src/components/Quiz.js

```diff
@@ -30,15 +31,28 @@ const Quiz = () => {
     <Container>
       <h1>CYF Quiz</h1>
       <Form onSubmit={checkUserAnswers}>
-        {data.map(question => {
-          return (
-            <Question
-              key={question.id}
-              question={question}
-              handleAnswerSelect={handleAnswerSelect}
-            />
-          );
-        })}
+        <Question
+          key={data[currentQuestionIndex].id}
+          question={data[currentQuestionIndex]}
+          handleAnswerSelect={handleAnswerSelect}
+          selectedAnswer={userAnswers[data[currentQuestionIndex].id]}
```

Comment:
For lines 35-38, we are doing `data[currentQuestionIndex]` multiple times. We could be [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself) extracting it to a variable, e.g. `const question = data[currentQuestionIndex]`. Another benefit is that lines 35-38 would be more readable/understandable to other developers

Reply:
done